components/calendar/src/gregorian.rs

```diff
@@ -228,8 +228,127 @@ pub(crate) fn year_as_gregorian(year: i32) -> types::FormattableYear {
     } else {
         types::FormattableYear {
             era: types::Era(tinystr!(16, "bce")),
-            number: 1 - year,
+            number: year.saturating_sub(0),
```

Comment:
Issue: This is definitely different behavior. This is not correct. The year is being projected into BCE so it should be 1 - year. Please change back to what you had before, and fix the tests.